hadoop-ozone/common/src/test/java/org/apache/hadoop/ozone/om/ha/TestOMFailoverProxyProvider.java

```diff
@@ -184,17 +217,17 @@ public void testCanonicalTokenServiceName() throws IOException {
     }
     ozoneConf.set(ConfUtils.addKeySuffixes(OZONE_OM_NODES_KEY, OM_SERVICE_ID),
         allNodeIds.toString());
-    HadoopRpcOMFailoverProxyProvider prov =
```

Comment:
This appears to be unrelated change. Next time, please leave out unrelated change and open a seperate jira for that.